prototype_source/pt2e_quant_ptq_static.rst

```diff
@@ -430,23 +436,43 @@ Convert the Calibrated Model to a Quantized Model
-   For example, here is how we plan to represent a quantized linear operator:
+   At this step, we currently have two representations that you can choose from, but what exact representation
+   we offer in the long term might change based on feedbacks from users.
+
+   * Q/DQ Representation (default)
+   Previous documentation for `representations <https://github.com/pytorch/rfcs/blob/master/RFC-0019-Extending-PyTorch-Quantization-to-Custom-Backends.md>`_ all quantized operators are represented as ``dequantize -> fp32_op -> qauntize``.
 
    .. code-block:: python
 
-     def quantized_linear(x_int8, x_scale, x_zero_point, weight_int8, weight_scale, weight_zero_point, bias_int32, bias_scale, bias_zero_point, output_scale, output_zero_point):
-         x_int16 = x_int8.to(torch.int16)
-         weight_int16 = weight_int8.to(torch.int16)
-         acc_int32 = torch.ops.out_dtype(torch.mm, torch.int32, (x_int16 - x_zero_point), (weight_int16 - weight_zero_point))
-         acc_rescaled_int32 = torch.ops.out_dtype(torch.ops.aten.mul.Scalar, torch.int32, acc_int32, x_scale * weight_scale / output_scale)
-         bias_int32 = torch.ops.out_dtype(torch.ops.aten.mul.Scalar, bias_int32 - bias_zero_point, bias_scale / output_scale))
-         out_int8 = torch.ops.aten.clamp(acc_rescaled_int32 + bias_int32 + output_zero_point, qmin, qmax).to(torch.int8)
-         return out_int8
-
-   For more details, please see:
-   `Quantized Model Representation <https://docs.google.com/document/d/17h-OEtD4o_hoVuPqUFsdm5uo7psiNMY8ThN03F9ZZwg/edit>`_.
+      def quantized_linear(x_int8, x_scale, x_zero_point, weight_int8, weight_scale, weight_zero_point, bias_fp32, output_scale, output_zero_point):
+          x_fp32 = torch.ops.quantized_decomposed.dequantize_per_tensor(
+                   x_i8, x_scale, x_zero_point, x_quant_min, x_quant_max, torch.int8)
+          weight_fp32 = torch.ops.quantized_decomposed.dequantize_per_tensor(
+                   weight_i8, weight_scale, weight_zero_point, weight_quant_min, weight_quant_max, torch.int8)
+          weight_permuted = torch.ops.aten.permute_copy.default(weight_fp32, [1, 0]);
+          out_fp32 = torch.ops.aten.addmm.default(bias_fp32, x_fp32, weight_permuted)
+          out_i8 = torch.ops.quantized_decomposed.quantize_per_tensor(
+          out_fp32, out_scale, out_zero_point, out_quant_min, out_quant_max, torch.int8)
+          return out_i8
+     
+     * Reference Quantized Model Representation (WIP, expected to be ready at end of August): we have special representation for selected ops (for example, quantized linear), other ops are represented as (dq -> float32_op -> q), and q/dq are decomposed into more primitive operators.
```

Comment:
```suggestion
     * Reference Quantized Model Representation (WIP, expected to be ready at end of August): we have special representation for selected ops (for example, quantized linear), other ops are represented as (``dq -> float32_op -> q``), and ``q/dq`` are decomposed into more primitive operators.
```

Reply:
The output is broken for this at the moment: https://docs-preview.pytorch.org/pytorch/tutorials/2530/prototype/pt2e_quant_ptq_static.html#convert-the-calibrated-model-to-a-quantized-model

Reply:
I see, I'll fix it